### PR TITLE
Allow users to send access request messages via Discord

### PR DESCRIPTION
**Pull request by vabold**

### CODE REVIEW

**Comment by vabold** [comment]
Ideally I'd still like to wait for the logo asset before merge, but it's not a blocker and that can easily be its own commit anyway.

File: srepanel/web/request_access.go
Comment (terorie) [inline srepanel/web/request_access.go:40]: These should be moved into a `discord` package and named after the API types, so we can reference them with e.g. `discord.EmbedAuthor`. Alternatively we can import https://github.com/bwmarrin/discordgo

Comment (terorie) [reply srepanel/web/request_access.go:40]: Sounds good to me

Comment (vabold) [reply srepanel/web/request_access.go]: Resolved in 22ccd92.

File: srepanel/common/types.go
Comment (terorie) [inline srepanel/common/types.go:6]: Looks fine for now. I suggest putting user information in the SQLite database long term, so the JWT doesn't grow without bounds.

File: srepanel/web/request_access.go
Comment (terorie) [inline srepanel/web/request_access.go:32]: I don't understand this TODO comment ... If the username doesn't exist in the UserManager, we can just add them before authorizing.

**Review by riptl — APPROVED** [review]

**Comment by riptl** [comment]
Thank you for the contribution @vabold 🎉

**Pull request merged**
